src/core/logger.c

```diff
@@ -18,6 +18,24 @@ static const char *loglevels[] = {
 
 static struct logger_properties logprops;
 
+FILE* open_logger_file(char *file_name) {
```

Comment:
IMO, I don't think we should support log files. Users can redirect stderr and stdout to a file if desired. @emersion, any opinion one way or the other?

Reply:
I added this, since it was simple and gave me a way to test the path expansion stuff. Can remove it, if wanted.

Reply:
This one is just my opinion and there may be some good reason to support file logging that I'm not thinking of, but I always expect services to log to stdout and stderr. That said, if xdpw isn't, it's probably due to a user's choice.

My two biggest concerns are around unneeded additional code surface area for bugs, and distros creating default configs that do unexpected things. Neither of those concerns seem likely.

Reply:
xdpw is usually started by xdg-desktop-portal. If it writes stuff to stdout/stderr, are they catched by journald?

Reply:
> xdpw is usually started by xdg-desktop-portal.

Technically xdpw is started via d-bus activation. xdp makes a d-bus call, which is associated with a particular systemd user service definition, which is automatically started. If you run `systemctl --user status xdg-desktop-portal-wlr.service`, you'll see that it is running and it's last few log lines will be abbreviated there.

Writing to a log file would break this behavior.

(All of this assumes systemd and journald. Technically this all can work without systemd. D-bus activation should still start xdpw, but I'm unfamiliar with this circumstance and haven't tested it. In either scenario, xdp wouldn't be the process parent)

Reply:
I would suggest, we just avoid this issue by removing the logfile option, until someone needs it.